openassessment/assessment/models/base.py

```diff
@@ -552,6 +598,28 @@ def get_median_score(scores):
             )
         return median_score
 
+    @staticmethod
+    def get_mean_score(scores):
+        """Calculate the mean score from a list of scores
+
+        Args:
+            scores (list): A list of int values. These int values
+                are reduced to a single value that represents the mean.
+
+        Returns:
+            (int): The mean score.
+
+        Examples:
+            >>> scores = [5, 6, 12, 16, 22, 53]
+            >>> Assessment.get_mean_score(scores)
+            19
+
+        """
+        total_criterion_scores = len(scores)
+        if total_criterion_scores == 0:
+            return 0
+        return math.ceil(sum(scores) / float(total_criterion_scores))
```

Comment:
I got an unexpected mean score:

![image](https://github.com/openedx/edx-ora2/assets/7556571/178a692f-4c0f-4283-982b-e046abe66eda)

(it's the second default rubric, so poor=0, fair=1, good=3, excellent=5):
1 + 0 + 3 + 1 + 5 = 10, 10/5 =2

Is it because there's no `2` in the rubric, so it rounds up to the next one `3`?

Reply:
This threw me for a loop. So I tried replicating the same scenario. Here's the criteria I used (defaults): 
Ideas: poor=0, fair=3, good=5
Content: poor=0, fair=1, good=3, excellent=5
Graded by 5 configuration
And got:
![image](https://github.com/openedx/edx-ora2/assets/64440265/9e4e777e-fc3b-4509-a0b8-56c33965dd5e)
 
While researching, I found that the assessments are [retrieved, ordered, and then truncated by the number of peer grades needed](https://github.com/openedx/edx-ora2/blob/master/openassessment/assessment/api/peer.py#L243-L248). Could you confirm that the number of peer grades configured was 5?

Reply:
I also added an extra test here with the exact values: https://github.com/openedx/edx-ora2/pull/2196/files#diff-7fe9b2681acaf897f2dd62a13553badf288fd6144832ce0562c4a58bd2640e7fR2381

Reply:
I also reproduced the same scenario with the graded-by-3 configurations and got 2 since the 1st three peers graded 1, 0, and 3 in that order. Please let me know if the submission order also affected your tests.

Reply:
Oh wow, submission order does affect the final score! That's annoying.

> Could you confirm that the number of peer grades configured while testing was 5?

Yes, 5 peer grades were required, and only 5 were received.

I used the default graded-by-5 configuration, with
Ideas: poor=0, fair=3, good=5
Content: poor=0, fair=1, good=3, excellent=5

Audit user received the same "Content" peer assessments as Honor, but in a different order, and so they got different scores. (Apologies about the "Ideas" assessments, I thought I did them all the same too, but apparently not.)

Audit received:
![image](https://github.com/openedx/edx-ora2/assets/7556571/e89458f4-61f7-4710-b0f0-53f05bb0e0cd)

Honor received:
![image](https://github.com/openedx/edx-ora2/assets/7556571/57e363d8-6024-4490-842b-15bc42a82939)